accord-core/src/main/java/accord/coordinate/Persist.java

```diff
@@ -86,12 +84,7 @@ public void onSuccess(Id from, ApplyReply reply)
                     if (!isDone)
                     {
                         isDone = true;
-                        Topologies topologies = tracker.topologies();
-                        Topology topology = topologies.forEpoch(txnId.epoch());
-                        int homeShardIndex = topology.indexForKey(route.homeKey());
-                        // we can persist only partially if some shards are already completed; in this case the home shard may not participate
-                        if (homeShardIndex >= 0)
-                            node.send(topology.get(homeShardIndex).nodes, to -> new InformDurable(to, topologies, route, txnId, executeAt, Majority));
+                        InformDurable.informHome(node, topologies, txnId, route, executeAt);
```

Comment:
we will now send as we fall back to the global topology yet the previous comment implies this isn't safe?

Reply:
it doesn't say it's unsafe, it says we might not have the information in our topologies. It's fine to inform the home shard (but perhaps redundant). I thought it was simpler to merge the different callers, but happy to split them again and introduce a `tryInformHome` (I did have this initially)

Reply:
its fine, I just wanted to get your feedback on it.  Redundant is fine by me, just making sure things were intentional (I have caused a lot of bugs due to refactoring missing subtle details)